electron/main/backend.ts

```diff
@@ -0,0 +1,85 @@
+import * as childProcess from "child_process";
+import { join, resolve } from "path";
+import treeKill from "tree-kill";
+
+const sendBackendLogToStudio = (win: Electron.BrowserWindow, data: string) => {
```

Comment:
`Electron.BrowserWindow` gives an error, need to `import { BrowserWindow } from 'electron'`